mednafen/pce_fast/vdc.cpp

```diff
@@ -1068,6 +1098,7 @@ void VDC_Init(int sgx)
 {
    unlimited_sprites = MDFN_GetSettingB("pce_fast.nospritelimit");
    correct_aspect = MDFN_GetSettingB("pce_fast.correct_aspect");
```

Comment:
There's probably some more in mednafen/settings.h.
I'll do some cleaning later.